init.lua

```diff
@@ -90,9 +75,10 @@ minetest.register_chatcommand("choosecase", {
 	func = function(name, params)
 		local lname = params:lower()
 		local worldpath = minetest.get_worldpath()
-		for iname, data in pairs(minetest.auth_table) do
+		local auth = minetest.get_auth_handler()
+		for iname, data in pairs(auth.iterate) do
```

Comment:
Pretty sure this should be `for iname, data in auth.iterate() do`

Reply:
you are correct